content/guides/container-supported-development/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Faster development and testing with container-supported development"
+linkTitle: Container-supported development
+summary: |
+  Containers don't have to be just for your app. Learn how to run your app's dependent services and other debugging tools to enhance your development environment.
+description: |
+  Use containers in your local development loop to develop and test faster… even if your main app isn't running in containers.
+levels: [beginner]
+params:
+  image: images/learning-paths/container-supported-development.png
+  skill: Beginner
```

Comment:
This was replaced by the `levels` taxonomy
```suggestion
```

Reply:
Whoops! Just missed that removal. Good catch.